### PR TITLE
MOTECH-2300: Improve tasks history UI

### PR DESCRIPTION
**Pull request by 0nat**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by sebbrudzinski** [comment]
test this please

File: modules/tasks/tasks/src/main/resources/webapp/js/controllers.js [hunk 1]
```diff
@@ -1112,7 +1112,7 @@
         });
     });
 
-    controllers.controller('TasksLogCtrl', function ($scope, Tasks, Activities, $stateParams, $filter, $http, ModalFactory, LoadingModal) {
+    controllers.controller('TasksLogCtrl', function ($scope, $rootScope, Tasks, Activities, $stateParams, $filter, $http, ModalFactory, LoadingModal, BootstrapDialogManager) {
```
Comment (ngraczewski) [inline modules/tasks/tasks/src/main/resources/webapp/js/controllers.js]: Why do we need $rootScope here?

**Comment by ngraczewski** [comment]
I belive we can adjust "Status" and "Retry" columns to take only as much space as they need.

**Comment by sebbrudzinski** [comment]
test this please

**Pull request merged**